src/openforms/conf/base.py

```diff
@@ -116,7 +116,7 @@
     # redis locks?
     "portalocker": {
         "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_PORTALOCKER', 'localhost:6379/0')}",
+        "LOCATION": f"redis://{config('CACHE_PORTALOCKER', 'localhost:6379/4')}",
```

Comment:
Was `4` chosen as a database number for a specific reason?

Reply:
it was the key closest to my middle finger :D